mysql-test/main/rpl_row_UUIDv7.test

```diff
@@ -0,0 +1,88 @@
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+
+--echo #
+--echo # Start of 11.5 tests
+--echo #
+
+--echo #
+--echo # MDEV-32637 Implement native UUID7 function
+--echo #
+
+--sync_slave_with_master
+
+# To ensure that the tables and procedures that the test
+# is going to insert are not already in the database, remove
+# them prior starting the test
+connection master;
+
+--disable_warnings
+DROP PROCEDURE IF EXISTS test.p1;
+DROP TABLE IF EXISTS test.t1;
+--enable_warnings
+
+# Start the actual test
+
+eval CREATE TABLE test.t1 (a INT, blob_column LONGBLOB, vchar_column VARCHAR(100), PRIMARY KEY(a)) engine=myisam;
+INSERT INTO test.t1  VALUES(1,UUIDv7(),UUIDv7());
+delimiter |;
+create procedure test.p1()
+begin
+  INSERT INTO test.t1  VALUES(2,UUIDv7(),UUIDv7());
+  INSERT INTO test.t1  VALUES(3,UUIDv7(),UUIDv7());
+end|
+delimiter ;|
+
+CALL test.p1();
+
+delimiter |;
+create function test.fn1(x int)
+       returns int
+begin
+       insert into t1 values (4+x,UUIDv7(),UUIDv7());
+       insert into t1 values (5+x,UUIDv7(),UUIDv7());
+       return 0;
+end|
+
+delimiter ;|
+# test both in SELECT and in INSERT
+--disable_ps2_protocol
+select fn1(0);
+--enable_ps2_protocol
+eval create table t2 (a int) engine=myisam;
+insert into t2 values(fn1(2));
+
+sync_slave_with_master;
+SHOW CREATE TABLE test.t1;
+
+# Dump the databases and so we can see if they match
+# Another method would be to use SELECT INTO OUTFILE on master,
+# then LOAD DATA INFILE in slave, and use a query to compare.
+# This would have the advantage that it would not assume
+# the system has a 'diff'
```

Comment:
Out of curiosity, are there platforms where `--diff_files` doesn't work? Windows?

There are a couple of long-existing tests which mention _both_ Windows-specific problems _and_ use `--diff-files`, but I haven't tested these on Windows myself…

- https://github.com/MariaDB/server/blob/11.3/mysql-test/main/loaddata.test#L528, https://github.com/MariaDB/server/blob/11.3/mysql-test/main/loaddata.test#L395
- https://github.com/MariaDB/server/blob/11.3/mysql-test/main/mysqltest.test#L2893, r

Reply:
Got it. I suspect the source comment is obsolete.